fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/ColumnRefOperator.java

```diff
@@ -69,6 +83,9 @@ public void setChild(int index, ScalarOperator child) {
     }
 
     public ColumnRefSet getUsedColumns() {
+        if (isLambdaArgument) {
+            return new ColumnRefSet();
+        }
```

Comment:
Return none-used directly will affect many optimization cases, which needs careful testing

Reply:
lambda arguments are different normal columnRefs, they are just used in current lambda functions, seems no existing rules can be applied to it.